packages/eas-cli/src/branch/queries.ts

```diff
@@ -14,24 +14,29 @@ import {
 
 export const BRANCHES_LIMIT = 50;
 
-export async function selectBranchFromPaginatedQueryAsync(
-  projectId: string,
-  promptTitle: string,
-  options: PaginatedQueryOptions
-): Promise<UpdateBranchFragment> {
-  if (options.nonInteractive) {
+export async function selectBranchOnAppAsync({
```

Comment:
i tried to standardize the names and format of these queries. since I'm coming back to work on this after some time away and I'm adding in more use-cases it is a little easier to understand good conventions. 

- the names are shortened to include the action (select/list) followed by the data source (on app / on branch)
- the arguments are now an object. 
  - this lends itself to many required args
  - it's easier to remove/add new arguments without unintentional breaking other uses
  - it's more explicit